protos/sources/vega/api/v1/core.proto

```diff
@@ -324,6 +324,10 @@ message Statistics {
   string epoch_start_time = 37;
   // Epoch expected end time
   string epoch_expiry_time = 38;
+  // Number of events in the last block
+  uint64 event_count = 39;
+  // Rate of events per second in total
```

Comment:
Don't quite understand this description, could you perhaps reword, or explain in a comment and I can provide a suggestion?

Reply:
`number_of_events_emitted_in_last_block / duration_in_seconds_last_block`

Reply:
Thank you!